src/mutating-policies.md

```diff
@@ -0,0 +1,129 @@
+### Mutating policies validation
+
+Since [Kubewarden controller](https://github.com/kubewarden/kubewarden-controller) version v0.4.5 and [Policy Server](https://github.com/kubewarden/policy-server) version v0.2.6, the `mutating` field impacts how mutated requests are validated. If you deploy a policy which mutates requests and the `mutating` field set to `true`, the requests will act as a validated ones.
```

Comment:
I would rephrase the paragraph to be clearer and shorted. Something of the like:
```suggestion
A mutating policy is one that can mutate requests sent to it. This means that it will change the objects when instantiating them. Those policies are written so they rebuild the request to be conformant with the policy. If you want them to instantiate the rebuilt request, that is, to make a policy behave as mutating, set `ClusterAdmissionPolicy.mutating` to `true`.

However, if you you just want the policy to validate the requests without mutating them, as a normal non-mutating policy, you can always set `ClusterAdmissionPolicy.mutating` to `false`.

Here's an example. When setting to true: (queue to psp-user-group example)
```

I would drop the "since kubewwarden-controller ..", we can assume that the docs target latest versions, if not we are going to have a lot of "since". In the future we could move to versioned docs instead.

Then later I would explain the behaviour with `ClusterAdmissionPolicy.mutating` set to `false`, and drop the "However, if you set the `mutating` field to false.." until after the example that sets it to false.

Reply:
Updated the first sentence ([bea3340](https://github.com/kubewarden/docs/pull/82/commits/bea3340a01e5f4998a5276438b2ab27064be6dbc))